src/pages/Members/containers/MemberFormContainer.tsx

```diff
@@ -56,12 +56,11 @@ const initialValues: CreateMemberInput | UpdateMemberInput = {
 
 interface Props {
   height: number;
-  id?: number;
+  id?: number | null;
```

Comment:
The id field is already optional. It seems the MemberFormContainer component should handle undefined logic inside it as it accepts an optional parameter.

Let me know what you think.

Reply:
makes sense, i have updated